09_json/willshen8/cmd/puppy-server/main_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseError(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	args = []string{"--wrongFlag"}
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("The code did not panic")
+		}
+	}()
+
+	main()
```

Comment:
`assert.Panics(main)`

Reply:
Do I just call `assert.Panics(main)` instead of `main`?